docs/customization/windows-gaming-hvm.md

```diff
@@ -1,169 +1,254 @@
-
-# Windows gaming HVM
+# Create a Gaming HVM
 
 Some information to configure a windows HVM for gaming. 
-This is not officially supported, just some community trial & errors
+This is not officially supported, just some community trial & errors.
+This doc is also hosted on
+https://neowutran.ovh/qubes/articles/gaming_windows_hvm.html
 
 ## References
 
 Everythings needed is referenced here
 
-- [Useful technical details](https://paste.debian.net/1043341/)
-- [Reddit thread of what is needed for GPU passthrough](https://www.reddit.com/r/Qubes/comments/9hp3e7/gpu_passthrough_howto/)
-- [Solution to have more than 3 GB of RAM in the Windows HVM](https://github.com/QubesOS/qubes-issues/issues/4321#issuecomment-423011787)
-- [Some old references](https://www.reddit.com/r/Qubes/comments/66wk4q/gpu_passthrough/)
+-   [Usefull technical details](https://paste.debian.net/1043341/)
```

Comment:
> Usefull

Useful

Reply:
The link `https://paste.debian.net/1043341/` is not available now. Please replace with something more reliable.